sky/clouds/aws.py

```diff
@@ -859,11 +859,11 @@ def _check_credentials(cls) -> Tuple[bool, Optional[str]]:
 
         # Fetch the AWS catalogs
         # pylint: disable=import-outside-toplevel
-        from sky.catalog import aws_catalog
+        from sky.catalog.data_fetchers import fetch_aws
 
         # Trigger the fetch of the availability zones mapping.
         try:
-            aws_catalog.get_default_instance_type()
+            fetch_aws.fetch_availability_zone_mappings()
```

Comment:
Seems we are calling into a function with cache in the previous function? This change might be wrong, less efficient for people with AWS enabled? https://github.com/skypilot-org/skypilot/blob/fc35b369a1dc5ff7d00cd42d7e768b8c3df4ba0c/sky/catalog/aws_catalog.py#L115-L182